src/main/resources/resources/logisim/strings/gui/gui.properties

```diff
@@ -684,17 +686,22 @@ argTtyOptionArgName = format
 argVersionOption = Display version number and exit
 invalidLocaleError = Specified locale is not supported.
 invalidLocaleOptionsHeader = Supported locales:
-loadMultipleError = The "--load" option can be specified only once.
-loadNeedsTtyError = The "--load" option works only in conjunction with "--tty".
+loadMultipleError = The "-load" option can be specified only once.
+loadNeedsTtyError = The "-load" option works only in conjunction with "-tty".
+saveMultipleError = The "-save" option can be specified only once.
+saveNeedsTtyError = The "-save" option works only in conjunction with "-tty".
 templateCannotReadError = No permission to read template file: %s
 templateMissingError = Template file does not exist: %s
-ttyFormatError = --tty requires at least one of the following: halt, speed, stats, table, tty
-ttyNeedsFileError = Using "--tty" requires a filename provided on command line.
+ttyFormatError = "-tty" requires at least one of the following: halt, speed, stats, table, tty, binary, hex, csv, tabs
+ttyNeedsFileError = Using "-tty" requires a filename provided on command line.
+
 #
 # start/TtyInterface.java
 #
 loadIoError = Error while reading image file
 loadNoRamError = No RAM was found for the "-load" option.
+saveIoError = Error while writing image file
+saveNoRamError = No RAM was found for the "-save" option.
```

Comment:
Long command line options start with a double dash `--` since clean-ups by @MarcinOrlowski last year.

Unfortunately, the documentation hasn't been updated yet and still mentions single dash for long command line options.

Reply:
I'm confused. Single dashes currently work for even for long names. What am I missing?

I am happy to retract the "extra" changes, or try to convert everything to the new standard. I was just trying not to take on too much in my first PR.

How do localization strings get created? Is there a way to let the translators know about them? (Is there a better way/place to have this conversation?)

Reply:
@MarcinOrlowski: Could you please comment regarding:

> I'm confused. Single dashes currently work for even for long names. What am I missing?

?

Reply:
Probably nothing. The fact that it works it most likely because of the library we use to handle CLI args, yet the most  used convention for CLI args is as mentioned and we want to keep all switches unified.

Reply:
I believe all these have been addressed in the latest commit.